### ISSUE TITLE
Default in switch case is removed

### ISSUE THREAD

**Issue by mick111** [nicklockwood/SwiftFormat#522]
When formatting this code, 
```swift
var a: Int = 0
switch a {
case 1: print(1)
case -1: print(-1)
default: print("beep"); break
}
```
the output is
```swift
var a: Int = 0
switch a {
case 1: print(1)
case -1: print(-1)
}
```
instead of something like
```swift
var a: Int = 0
switch a {
case 1: print(1)
case -1: print(-1)
default: print("beep")
}
```
Making code not compiling (Switch must be exhaustive). 

No specific configuration changed.

`./SwiftFormat-0.43.4/CommandLineTool/swiftformat main.swift`

**Comment by nicklockwood** [comment]
@mick111 thanks for reporting - I'll get it fixes asap. In the meantime, manually removing the semicolon will fix it.

**Comment by nicklockwood** [comment]
@mick111 fixed in 0.43.5.
